index.js

```diff
@@ -11,14 +11,21 @@ if (other) {
   }
 } else
 if (osx) {
-  var regPath = '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
+  var osxSuffix = '/Contents/MacOS/Google Chrome'
+  var regPath = '/Applications/Google Chrome.app' + osxSuffix
   var altPath = require('userhome')(regPath.slice(1))
+  var mdFindCmd = 'mdfind \'kMDItemDisplayName == "Google Chrome" && kMDItemKind == Application\''
 
-  module.exports = fs.existsSync(regPath)
-    ? regPath
-    : altPath
+  if (fs.existsSync(regPath)) {
+    module.exports = regPath
+  } else if (fs.existsSync(altPath)) {
+    module.exports = altPath
+  } else {
+    var foundPath = require('child_process').execSync(mdFindCmd, { encoding: 'utf8' })
```

Comment:
I think it's definitely worth it since the internals require async to happen to yield a correct result. Ideally I'd be for returning a Promise and calling a callback to make everyone happy. That said having a Promise polyfill might be overkill, maybe.